lib/matplotlib/ticker.py

```diff
@@ -1426,9 +1423,9 @@ def set_params(self, **kwargs):
         if 'integer' in kwargs:
             self._integer = kwargs['integer']
         if self._integer:
-            self._steps = [n for n in self._steps if divmod(n, 1)[1] < 0.001]
+            self._steps = [n for n in self._steps if _divmod(n, 1)[1] < 0.001]
 
-    def bin_boundaries(self, vmin, vmax):
```

Comment:
Note tha the behavior of `_raw_ticks` is slightly different from `bin_boundaries` (namely it handles properly floating-point inaccuracies), so given that strict backwards compatibility has been broken I think it's a good opportunity to make it private.  If you prefer I can leave the (now unused) old implementation of `bin_boundaries` in, with a deprecation warning.